src/graphics/webgl/webgl-shader.js

```diff
@@ -13,6 +13,14 @@ import { getProgramLibrary } from '../get-program-library.js';
  *
  * @ignore
  */
+
+const _vertexShaderBuiltins = [
+    'gl_VertexID',
+    'gl_InstanceID',
+    'gl_DrawID',
+    'gl_BaseVertex',
+    'gl_BaseInstance'
+];
```

Comment:
Move to above the JSDoc for the class. Or it the docs will be disconnected from the code.

Reply:
how did I miss this :(